utils/web3_utils.py

```diff
@@ -21,21 +22,11 @@
 w3_scroll = Web3(Web3.HTTPProvider(SCROLL_NODE_URL))
 
 W3_BY_CHAIN = {
-    "eth": {
-        "w3": w3,
-    },
-    "arb": {
-        "w3": w3_arb,
-    },
-    "mantle": {
-        "w3": w3_mantle,
-    },
-    "blast": {
-        "w3": w3_blast,
-    },
-    "scroll": {
-        "w3": w3_scroll,
-    },
+    Chain.ETHEREUM: w3,
```

Comment:
The `Chain(Enum)` values did not correspond to the fields in `W3_BY_CHAIN` requiring (a) duplicate declarations of chain constants and (b) making chain declarations more typo prone. Modified the enum so that it can be directly used by the `WS_BY_CHAIN` dict.